snippets/allcolors/allcolors.js

```diff
@@ -86,7 +86,7 @@
     return "background:" + color + ";color:" + color + ";border:1px solid #333;";
   };
 
-  console.group("All colors used in elements on the page: " + window.location.href);
+  console.group("Total colors used in elements on the page: " + window.location.href + " are " +allColorsSorted.length);
```

Comment:
Uber-nit: can you please add an extra space in between the + and the allColorsSorted.length?